assignment6.js

```diff
@@ -0,0 +1,24 @@
+const prompt=require("prompt-sync")();
+
+function check_leapyear(){
+    const year=prompt("Enter the year");
+    if(year!=null)
```

Comment:
in this type always better to use guard statement like
if (year == null)
   return false

Reply:
updated